doc/admin/pool_operations.md

```diff
@@ -177,13 +177,34 @@ with one ACE per line.
 
 ### Modifying a pool's ACL
 
+For all of these commands using an ACL file, the ACL file must be in the format
+noted above for pool creation.
+
+#### Overwriting the ACL
```

Comment:
Is deleting ACL / entries expected to be achieved through overwrite (by having an existing entry not present in the new file presented to overwrite), or are there further plans to implement a different command for this?

Reply:
Delete-acl is a separate command that is in fact getting ready to for a PR after this one goes in. :)